src/index.ts

```diff
@@ -199,20 +197,16 @@ export class GoogleToken {
       }
       case '.p12':
       case '.pfx': {
-        // NOTE:  The loading of `google-p12-pem` is deferred for performance
-        // reasons.  The `node-forge` npm module in `google-p12-pem` adds a fair
-        // bit time to overall module loading, and is likely not frequently
-        // used.  In a future release, p12 support will be entirely removed.
-        if (!getPem) {
-          getPem = (await import('google-p12-pem')).getPem;
-        }
-        const privateKey = await getPem(keyFile);
-        return {privateKey};
+        throw new ErrorWithCode(
+          '*.p12 certificates are not supported. ' +
```

Comment:
NB: should we add a version number for when it was last supported?

Reply:
Added.